tuning-primer.sh

```diff
@@ -229,10 +237,11 @@ function second_login_failed()
   cecho "Could not auto detect login info!"
   cecho "Found potential sockets: $found_socks"
   cecho "Using: $socket" red
-  read -p "Would you like to provide a different socket?: [y/N] " REPLY
+  read -p "Would you like to provide a different socket/host?: [y/N] " REPLY
```

Comment:
Host and socket are mutually exclusive*, and the UX of the prompts that follow is strange as a result.  Change `y`/`N` to something like `s`/`h`/`N`.  Or ask host and socket as two separate questions; both approaches strike me as valid.

_*: Excepting the weird special case of `-h localhost`, which is a total disaster unto itself._